src/Streamly/Internal/Data/Array/Prim/MutTypesInclude.hs

```diff
@@ -266,7 +266,7 @@ fromStreamDArraysOf n (D.Stream step state) = D.Stream step' (GroupStart state)
         when (n <= 0) $
             -- XXX we can pass the module string from the higher level API
             error $
-            "Streamly.Internal.Data.Array.Storable.Foreign.Mut.Types.fromStreamDArraysOf: the size of " ++
+            "Streamly.Internal.Data.Array.Foreign.Mut.Types.fromStreamDArraysOf: the size of " ++
```

Comment:
This should be `...Prim...`. Since its a common module you can maybe write `...-Array.Prim.(Pinned).Mut-...`